draft-wang-ppm-dap-taskprov.md

```diff
@@ -56,69 +60,302 @@ collector). Thus it is up to each deployment of DAP to securely implement its
 own mechanism.
 
 This document describes a mechanism for configuring tasks that may be useful in
-many deployments.
+many deployments. The goal of this mechanism is to add transparency to the
+task provision process, and define a task provision method that does not rely
+on deployment specific leader-helper out-of-band agreement.
```

Comment:
Vanilla DAP does not mandate some sort of "leader-helper" agreement. Rather, the means by which parties are configured is not specified.